web/src/components/filters-toolbar.tsx

```diff
@@ -155,10 +154,13 @@ export const FiltersToolbar: React.FC<FiltersToolbarProps> = ({
       if (!selectedFilterColumn) {
         return { err: t('Column must be selected') };
       } else if (_.isEmpty(value)) {
+        if (selectedFilterColumn.filterType === FilterType.NAMESPACE) {
```

Comment:
Should we do that for any enriched columns ?

Reply:
```suggestion
        if (selectedFilterColumn.filterType === FilterType.NAMESPACE || selectedFilterColumn.filterType === FilterType.K8S_NAMES) {
```

Reply:
that was my plan initially, but on logql side it's more complicated, I want to do that on a next PR, see my comment in JIRA: https://issues.redhat.com/browse/NETOBSERV-223?focusedCommentId=19941741&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-19941741
I need to dig more in the loki query builder for that. Whereas for labels like namespace, it works out of the box.